src/collection.js

```diff
@@ -31,6 +31,18 @@ function createOptionsObject(target, userOptions) {
   return userOptions;
 }
 
+//Get the rendered center 
+function getRenderedCenter(target){
+  let pos = target.renderedPosition();
+  let offsetX = target.renderedWidth() / 2;
```

Comment:
This should use `getDimensions()` so the user's definition of the element's dimensions is applied here.